package.json

```diff
@@ -54,7 +54,7 @@
     "start:dev": "npm run remove-rcue; sh -ac 'source ./.env.upstream; npm run start:kiali'",
     "start:kiali": "npm run copy-fonts && npm run copy-img && npm run build-css && REACT_APP_VERSION=$npm_package_version REACT_APP_NAME=$npm_package_name REACT_APP_GIT_HASH=$(git rev-parse HEAD) react-scripts start",
     "start:prod": "npm run restore-rcue; sh -ac 'source ./.env.downstream; npm run start:kiali'",
-    "test": "npm run build-css && TEST_RUNNER=1 react-scripts test --env=jsdom __tests__",
+    "test": "npm run build-css && TEST_RUNNER=1 craco test --env=jsdom __tests__",
```

Comment:
Why this change?

Reply:
We are using (at least for this version) a webcomponent. When running the tests, the webcomponent is imported and called and it leads to a "customElements is not defined".
Using craco helps to override the import of the web element so tests can run.
Any alternate suggestion is welcomed

Reply:
I think we do something like that for `SVGPathElement`, or `SVGElement` (not at the computer right now to look).

A reference that might help (it did when I found this problem): https://github.com/jsdom/jsdom/issues/1423

Reply:
Thanks, I was able to do the same so I will remove the craco override